app/views/mailers/mailshots_mailer/challenge_12in23_launch.text.erb

```diff
@@ -0,0 +1,24 @@
+Hi <%= @user.handle %>,
+
+Happy new year! For 2023, we’ve decided to do something a bit different. We're encouraging everyone to try 12 different languages throughout the year! We're calling this the #12in23 Challenge: https://exercism.org/challenges/12in23
+
+Exploring different languages is a great way of opening your eyes to new concepts and paradigms and broadening your programming horizons. We’ve got over 60 languages ranging from the old (looking at you, Cobol) to the new and shiny (hello, Unison!) to the esoteric (everyone’s favourite: Prolog). We’ve got functional languages, OOPs languages, more lisps than you can shake a fist at, and some downride odd stuff like vimscript! There’s loads to choose from 🙂
```

Comment:
I'm not entirelt sure about "downright odd", that might be offensive to people that like vimscript (and there are "odder" languages I think)

```suggestion
Exploring different languages is a great way of opening your eyes to new concepts and paradigms and broadening your programming horizons. We’ve got over 60 languages ranging from the old (looking at you, Cobol) to the new and shiny (hello, Unison!) to the esoteric (everyone’s favourite: Prolog). We’ve got functional languages, OOPs languages, more lisps than you can shake a fist at, and some downright niche stuff like vimscript! There’s loads to choose from 🙂
```

Reply:
This comment likely applies to the other template too

Reply:
Nice.